src/schema/clan/README.md

```diff
@@ -0,0 +1,23 @@
+# Clan
+
+Clans are a permanent kind of party with additional features.
+
+A clan has a profil which includes several information like tag, name, description. The following use-case diagrams gives an overview about the topics which has to be handled with this schema:
+
+### Manage clan
+
+Every user is allowed to create a clan with [clan/create](#create). Then he is the clan leader. The clan leader is the only one who is allowed to delete the clan with the request [clan/delete](#delete).
```

Comment:
Why do we care about specifying behavior? Is this, and everything below this line, something _strictly_ required by the messaging contract or just an example of a possible implementation?